mappings.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0"?>
+<mappings>
+  <mapping>
+    <path>~/navigation/position/latitude</path>
+    <sentence>
+      <id>RMC</id>
+      <fieldset type="coordinate">
+        <field>2</field>
+        <field>3</field>
+      </fieldset>
+      <filter type="if">
+        <field>1</field>
+        <value transform="uppercase">A</value>
+      </filter>
+    </sentence>
+  </mapping>
+
+  <mapping>
+    <path>~/navigation/position/longitude</path>
```

Comment:
I tend to agree, I would have preferred a JSON mapping that is less cumbersome. This is based on Tim's mapping, and the general idea is quite smart: each `<mapping>` has one path, and one or more sentences and PGNs.. and in the future perhaps other mappings.